src/braket/device_schema/quera/quera_ahs_paradigm_properties_v1.py

```diff
@@ -0,0 +1,328 @@
+    Attributes:
+        c6Coefficient (Decimal): Rydberg-Rydberg C6 interaction coefficient
+        timeMax (Decimal): Maximum duration of Rydberg drive
+        rydbergGlobal: Rydberg Global
+        rydbergLocal: Rydberg Local
+    """
+
+    c6Coefficient: Decimal
+    timeMax: Decimal
+    rydbergGlobal: RydbergGlobal
+    rydbergLocal: RydbergLocal
+
+
+class PerformanceRydbergGeometry(BaseModel):
+    """
+    Performance of Rydberg Geometry
+    Attributes:
+        positionErrorAbs (Decimal): Error between target and actual site position
+    """
+
+    positionErrorAbs: Decimal
+
+
+class PerformanceRydbergGlobal(BaseModel):
+    """
+    Performance of Rydberg Global
+    Attributes:
+        rabiFrequencyErrorRel (Decimal): random error in the Rabi frequency, relative
+        rabiFrequencyHomogeneity (Decimal): fractional RMS Rabi frequency
+            non-uniformity across the user area
+        detuningErrorAbs (Decimal): random error in the detuning, absolute
+        phaseErrorAbs (Decimal): random error in the phase, absolute
+        omegaTau (Decimal): Product of Rabi frequency and Rabi decay time
+            for a single qubit under resonant Rydberg drive - average over
+            field of view (no local detuning applied)
+        singleQubitFidelity (Decimal): Single qubit fidelity of a pi(3.14)  pulse
+        twoQubitFidelity (Decimal): Fidelity of two-qubit entangled state
+    """
+
+    rabiFrequencyErrorRel: Decimal
+    rabiFrequencyHomogeneity: Decimal
+    detuningErrorAbs: Decimal
+    phaseErrorAbs: Decimal
+    omegaTau: Decimal
+    singleQubitFidelity: Decimal
+    twoQubitFidelity: Decimal
+
+
+class PerformanceRydbergLocal(BaseModel):
+    """
+    Performance of Rydberg Local
+    Attributes:
+        detuningDynamicRange (Decimal): Dynamic range over which the
+            specified local detuning accuracy is preserved, relative to max detuning used.
+            E.g. if the largest local detuning is 2*pi(3.14) *37.0e6 rad/s, detuning
+            accuracy (error) is preserved for other sites down to 2*pi(3.14) *3.7e6
+        detuningErrorRel (Decimal):random local detuning error, relative to the given local detuning
+        detuningHomogeneity (Decimal): worst-case fractional RMS detuning non-uniformity of
+            the detuning pattern, relative to the max detuning used in the detuning pattern;
+            applies over the specified dynamic range only
+        detuningScaleErrorRel (Decimal): Error on the global scaling factor for local
+            detuning, relative
+        darkErrorRete (Decimal): Error rate induced by local detuning pattern under Rydberg
+            drive for qubits maximally detuned by the local detuning pattern (scattering)
+    """
+
+    detuningDynamicRange: Decimal
+    detuningErrorRel: Decimal
+    detuningHomogeneity: Decimal
+    detuningScaleErrorRel: int
+    darkErrorRete: Decimal
+
+
+class PerformanceRydberg(BaseModel):
+    """
+    Rydberg Performance
```

Comment:
A better description would be: "Parameters determining the limitations the Rydberg simulator"